Cargo.toml

```diff
@@ -50,7 +51,8 @@ name = "key"
 # debug = true
 
 [features]
-default = []
+default = ["std"]
+std = []
```

Comment:
shouldn’t this enable the std features on the crates that got it disabled above?